integration-tests/smoke/ccip/ccip_batching_test.go

```diff
@@ -1,4 +1,4 @@
-package smoke
+package ccip
```

Comment:
Wrong package names on these files, not sure how this didn't throw a build error before.

Reply:
It is allowed, but I'm not sure I've ever seen explanations for cases other than `main` or `*_test` :thinking:

Reply:
Oh, versioned packages is another case. e.g. math/rand/v2